driver/pace/driver/comm.py

```diff
@@ -46,15 +46,11 @@ class CommConfig:
 
     config: CreatesComm
     type: str = "mpi"
-    registry: ClassVar = {}
+    registry: ClassVar = Registry(default_type="mpi")
```

Comment:
If I understand what you mean, I think I can update it to ClassVar[Registry] to specify the type. I do need to initialize and set the registry here, since it's a class variable.